nbgrader/apps/fetchfeedbackapp.py

```diff
@@ -0,0 +1,95 @@
+# coding: utf-8
+
+from traitlets import default
+
+from .baseapp import NbGrader, nbgrader_aliases, nbgrader_flags
+from ..exchange import Exchange, ExchangeFetchFeedback, ExchangeError
+
+
+aliases = {}
+aliases.update(nbgrader_aliases)
+aliases.update({
+    "timezone": "Exchange.timezone",
+    "course": "Exchange.course_id",
+})
+
+flags = {}
+flags.update(nbgrader_flags)
+
+
+class FetchFeedbackApp(NbGrader):
+
+    name = u'nbgrader-fetch-feedback'
+    description = u'Fetch feedback for an assignment from the nbgrader exchange'
+
+    aliases = aliases
+    flags = flags
+
+    examples = """
+        Fetch feedback for an assignment that an instructor has released. MORE INFO NEEDED
+        For the usage of students.
+
+        You can run this command from any directory, but usually, you will have a
+        directory where you are keeping your course assignments.
+
+        To fetch an assignment by name into the current directory:
+
+            nbgrader fetch assignment1
+
+        To fetch an assignment for a specific course, you must first know the
+        `course_id` for your course.  If you don't know it, ask your instructor.
+        Then, simply include the argument with the '--course' flag.
+
+            nbgrader fetch assignment1 --course=phys101
+
+        This will create an new directory named `assignment1` where you can work
+        on the assignment. When you are done, use the `nbgrader submit` command
+        to turn in the assignment.
+        """
+
+    @default("classes")
+    def _classes_default(self):
+        classes = super(FetchFeedbackApp, self)._classes_default()
+        classes.extend([Exchange, ExchangeFetchFeedback])
+        return classes
+
+    def _load_config(self, cfg, **kwargs):
+        if 'FetchApp' in cfg:
```

Comment:
I don't think you need this here, as this is a deprecation warning for the regular (assignment) fetch functionality.

Reply:
Removed method in https://github.com/danielmaitre/nbgrader/pull/8